test/app/onboard/App/Slides/index.test.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 
-import { setupComponent } from '../../../../componentSetup'
+import { user, screen, setupComponent } from '../../../../componentSetup'
 import Slides from '../../../../../app/onboard/App/Slides'
 
 it('completes when the user clicks close', async () => {
   const onComplete = jest.fn()
-  const { user, getByRole } = setupComponent(<Slides slide={6} onComplete={onComplete} />)
+  setupComponent(<Slides slide={6} onComplete={onComplete} />)
 
-  await user.click(getByRole('button', { name: 'Close' }))
+  await user.click(screen.getByRole('button', { name: 'Close' }))
```

Comment:
I think this is the old method of using `user-event`, ie pre v14. here is the more recent documentation: https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent